db/initdb.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS videos CASCADE;
+
+CREATE TABLE videos(
+    name VARCHAR,
+    src VARCHAR
+);
+
+INSERT INTO videos (name,src) VALUES ('The Coding Train', 'https://www.youtube.com/watch?v=HerCR8bw_GE');
```

Comment:
Make sure we have "enable newline at end of files" turned on in your favourite IDE.

Also note creating the initdb script is one of the exercises, so not sure if we want to add it to the initial project